templates/build_template.go

```diff
@@ -460,7 +477,7 @@ aosp_repo_modifications() {
       print "  ";
       print "  <remote name=\"github\" fetch=\"https://github.com/RattlesnakeOS/\" revision=\"" ANDROID_VERSION "\" />";
       print "  <remote name=\"fdroid\" fetch=\"https://gitlab.com/fdroid/\" />";
-      print "  <remote name=\"prepare-vendor\" fetch=\"https://github.com/anestisb/\" revision=\"master\" />";
+      print "  <remote name=\"prepare-vendor\" fetch=\"https://github.com/cernekee/\" revision=\"master\" />";
```

Comment:
I just forked from your android-prepare-vendor and put it here: https://github.com/RattlesnakeOS/android-prepare-vendor. Could you instead point that here? Just want to avoid adding additional people that need to be trusted when building RattlesnakeOS.

Reply:
Done. (Untested.)